app/controllers/uploaded_files_controller.rb

```diff
@@ -32,9 +32,15 @@ def edit
   end
 
   def create
-    @uploaded_file = UploadedFile.new(description: uploaded_file_params['description'])
-    @uploaded_file.user = current_user
-    @uploaded_file.actual_file = uploaded_file_params['actual_file']
+    if uploaded_file_params['actual_file'].nil?
+      @uploaded_file = UploadedFile.new(actual_file_file_name: 'default.png')
```

Comment:
If the user has not uploaded a file, I think we should show an error indicating that - rather than default to setting the file to a default file.